src/backend/gog/games.ts

```diff
@@ -383,11 +384,14 @@ class GOGGame extends Game {
           ? wineExec.replaceAll("'", '')
           : wineExec
 
-      wineFlag.push(
-        ...(wineType === 'proton'
-          ? ['--no-wine', '--wrapper', `'${wineBin}' run`]
-          : ['--wine', wineBin])
-      )
+      if (wineType === 'proton') {
+        wineFlag.push(...['--no-wine', '--wrapper', `'${wineBin}' run`])
```

Comment:
Same suggestion to remove these `...[____]` pushes

```suggestion
        wineFlag.push('--no-wine', '--wrapper', `'${wineBin}' run`)
```